content/learn/03.programming/06.memory-guide/memory-guide.md

```diff
@@ -10,9 +10,9 @@ tags:
 author: 'Arduino, José Bagur, Taddy Chung'
 ---
 
-A microcontroller unit (also known as a MCU) is an integrated circuit (IC), typically used to perform specific applications or tasks. Usually, this type of IC gathers information or data from its surroundings, process it, and generates specific outputs according to the gathered data. Microcontrollers today are everywhere; they are an essential part of modern embedded systems that can be found practically everywhere in our world, from smartwatches to electric vehicles; they are even on the Martian surface right now. 
+A microcontroller unit (also known as an MCU) is an integrated circuit (IC), typically used to perform specific applications or tasks. Usually, this type of IC gathers information or data from its surroundings, processes it, and generates specific outputs according to the gathered data. Microcontrollers today are everywhere; they are an essential part of modern embedded systems that can be found practically everywhere in our world, from smart watches to electric vehicles; they are even on the Martian surface right now. 

```

Comment:
Should not it be `a` or just `also known as MCU`? 🤔

Reply:
I think it would be `also known as MCU` as it's a designation as overall. 🤔 Although I believe it would be better in my opinion to just state as `A microcontroller unit (MCU)` . How do you see it?

Reply:
I'll resolve this and make the changes within this 👍

Reply:
It is fine for me :)